nixos/modules/services/backup/restic.nix

```diff
@@ -333,6 +333,7 @@ in
               backup.rcloneConfig);
             path = [ pkgs.openssh ];
             restartIfChanged = false;
+            wants = [ "network-online.target" ];
```

Comment:
Should also set `after = [ "network-online.target" ];`

Reply:
added